v2/arangodb/collection_documents_update.go

```diff
@@ -68,6 +68,8 @@ type CollectionDocumentUpdateOptions struct {
 	WithWaitForSync *bool
 	NewObject       interface{}
 	OldObject       interface{}
+	// RefillIndexCaches if set to true then refills the in-memory index caches.
+	RefillIndexCaches *bool
```

Comment:
In V1 you have added it to the Create struct - here to Update one
Shouldn't it be a part of: https://github.com/arangodb/go-driver/blob/0783ea0ea35485f4115a58c9f487e9b3040c43f6/v2/arangodb/collection_documents_create.go#L108-L108 ?

Reply:
In v2 `RefillIndexCaches`is in:
- CollectionDocumentCreateOptions
- CollectionDocumentDeleteOptions
- CollectionDocumentUpdateOptions